src/tigerbeetle/cli.zig

```diff
@@ -377,6 +374,36 @@ pub fn parse_args(allocator: std.mem.Allocator, args_iterator: *std.process.ArgI
             };
         },
         .start => |start| {
+            // Allowlist of stable flags. --development will disable automatic multiversion
+            // upgrades too, but the flag itself is stable.
+            const stable_args = .{
+                "addresses",   "cache_grid",   "positional",
+                "development", "experimental",
+            };
+            inline for (std.meta.fields(@TypeOf(start))) |field| {
+                const stable_field = comptime for (stable_args) |stable_arg| {
```

Comment:
Assert that there's no typos in `stable_args`:

```suggestion
                const stable_field = comptime for (stable_args) |stable_arg| {
                    assert(std.meta.fieldIndex(@TypeOf(start), stable_arg) != null);
```